server/db/helper.js

```diff
@@ -67,9 +69,27 @@ const getCourse = (id) => {
     });
 };
 
+const getUserBadges = (userId) => UserBadge.findAll({
+  where: {
+    idUser: userId
+  },
+
+  attributes: ['idBadge']
+})
+.then((badgesId) => Badge.findAll({
```

Comment:
The values in sequelize aren't returned in an array but an object I believe, so console logging the value might help with the issue you are seeing